src/util-proto-name.c

```diff
@@ -184,6 +188,264 @@ const char *known_proto[256] = {
+    NULL,         /* 0x6B: 107 - Active Networks */
+    "ipcomp",     /* 0x6C: 108 - IP Payload Compression Protocol	RFC 3173 */
+    NULL,         /* 0x6D: 109 - Sitara Networks Protocol */
+    NULL,         /* 0x6E: 110 - Compaq Peer Protocol */
+    NULL,         /* 0x6F: 111 - IPX in IP */
+    "vrrp", /* 0x70: 112 - Virtual Router Redundancy Protocol, Common Address Redundancy Protocol
+            (not IANA assigned)	VRRP:RFC 3768 */
+    NULL,   /* 0x71: 113 - PGM Reliable Transport Protocol	RFC 3208 */
+    NULL,   /* 0x72: 114 - 0-hop protocol */
+    "l2tp", /* 0x73: 115 - Layer Two Tunneling Protocol Version 3	RFC 3931 */
+    NULL,   /* 0x74: 116 - D-II Data Exchange (DDX) */
+    NULL,   /* 0x75: 117 - Interactive Agent Transfer Protocol */
+    NULL,   /* 0x76: 118 - Schedule Transfer Protocol */
+    NULL,   /* 0x77: 119 - SpectraLink Radio Protocol */
+    NULL,   /* 0x78: 120 - Universal Transport Interface Protocol */
+    NULL,   /* 0x79: 121 - Simple Message Protocol */
+    NULL,   /* 0x7A: 122 - Simple Multicast Protocol	draft-perlman-simple-multicast-03 */
+    NULL,   /* 0x7B: 123 - Performance Transparency Protocol */
+    "isis", /* 0x7C: 124 - over IPv4	Intermediate System to Intermediate System (IS-IS) Protocol
+            over IPv4	RFC 1142 and RFC 1195 */
+    NULL,   /* 0x7D: 125 - Flexible Intra-AS Routing Environment */
+    NULL,   /* 0x7E: 126 - Combat Radio Transport Protocol */
+    NULL,   /* 0x7F: 127 - Combat Radio User Datagram */
+    NULL,   /* 0x80: 128 - Service-Specific Connection-Oriented Protocol in a Multilink and
+               Connectionless Environment	ITU-T Q.2111 (1999) */
+    NULL,   /* 0x81: 129 -  */
+    NULL,   /* 0x82: 130 - Secure Packet Shield */
+    NULL,   /* 0x83: 131 - Private IP Encapsulation within IP	Expired I-D
+               draft-petri-mobileip-pipe-00.txt */
+    "sctp", /* 0x84: 132 - Stream Control Transmission Protocol	RFC 4960 */
+    "fc",   /* 0x85: 133 - Fibre Channel */
+    NULL,   /* 0x86: 134 - Reservation Protocol (RSVP) End-to-End Ignore	RFC 3175 */
+    "mobility-header", /* 0x87: 135 - Header	Mobility Extension Header for IPv6	RFC 6275 */
+    "udplite",         /* 0x88: 136 - Lightweight User Datagram Protocol	RFC 3828 */
+    "mpls-in-ip",      /* 0x89: 137 - Multiprotocol Label Switching Encapsulated in IP	RFC 4023,
+                                RFC      5332 */
+    NULL,              /* 0x8A: 138 - MANET Protocols	RFC 5498 */
+    "hip",             /* 0x8B: 139 - Host Identity Protocol	RFC 5201 */
+    "shim6",           /* 0x8C: 140 - Site Multihoming by IPv6 Intermediation	RFC 5533 */
+    "wesp",            /* 0x8D: 141 - Wrapped Encapsulating Security Payload	RFC 5840 */
+    "rohc",            /* 0x8E: 142 - Robust Header Compression	RFC 5856 */
+    /* no aliases for 142-255 */
+};
+
+typedef struct ProtoNameHashEntry_ {
+    const char *name;
+    const char *alias;
+    uint8_t number;
+} ProtoNameHashEntry;
+
+HashTable *proto_ht;
```

Comment:
I generally make such globals `static` and initialize them to `NULL`. Should we do that here too?

Reply:
Yes -- will update.